tools/fuse-waked/fuse-waked.cpp

```diff
@@ -697,6 +697,19 @@ static int wakefuse_unlink_trace(const char *path)
 	return out;
 }
 
+// Check if the directory has children that were written and not yet deleted.
+// Note: we don't check the 'files_visible' list as when the directory itself or
+// a file within that directory is visible then is_writable() would have already failed.
+static bool has_written_children(const std::string& dir, Job& job) {
```

Comment:
```suggestion
static bool has_written_children(const std::string& dir, const Job& job) {
```

I think this can be a const reference?